wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java

```diff
@@ -201,7 +201,11 @@ public static boolean getBooleanProperty(String name, boolean defaultValue) {
   /**
    * Common initialization for all robot programs.
    */
-  public static void initializeHardwareConfiguration() {
+  public synchronized static void initializeHardwareConfiguration() {
```

Comment:
Got a complaint:
[ant:checkstyle] [ERROR] /data/home/jwhite/w/allwpilib/wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java:204:23: 'static' modifier out of order with the JLS suggestions. [ModifierOrder]
switching to 'static synchronized' eliminated the error.